specification/elasticsan/resource-manager/Microsoft.ElasticSan/stable/2023-01-01/elasticsan.json

```diff
@@ -1150,9 +1464,6 @@
     },
```

Comment:
Do you need to customize these? I guess its probably too late now from point of view of breaking changes. There are standard typedefs for these in common-types though. Maybe you could inherit them or something. Not really important, consdier doing it later?